src/Components/SceneRefreshConfigurator/SceneRefreshConfigurator.tsx

```diff
@@ -0,0 +1,268 @@
+            sceneId
+        );
+        setSelectedPollingStrategy(option.data);
+        ViewerConfigUtility.setPollingStrategy(config, sceneId, option.data);
+    };
+    const onRateChange = (
+        _event: React.FormEvent<HTMLDivElement>,
+        option: IDropdownOption<number>
+    ) => {
+        logDebugConsole(
+            'info',
+            `Updating selected polling rate to ${option.data} {config, sceneId}`,
+            config,
+            sceneId
+        );
+        setSelectedPollingRate(option.data);
+        ViewerConfigUtility.setPollingRate(config, sceneId, option.data);
+    };
+
+    const getLabelRenderFunction = (tooltipText: string) => {
+        return (
+            props?: IDropdownProps,
+            defaultRender?: (props?: IDropdownProps) => JSX.Element | null
+        ) => {
+            return (
+                <Stack horizontal verticalAlign={'center'}>
+                    {defaultRender(props)}
+                    <TooltipCallout
+                        content={{
+                            calloutContent: tooltipText,
+                            buttonAriaLabel: tooltipText
+                        }}
+                    />
+                </Stack>
+            );
+        };
+    };
+
+    // side effects
+    useEffect(() => {
+        // ensure that state never gets out of sync with the config
+        setSelectedPollingStrategy(pollingConfig.pollingStrategy);
+    }, [pollingConfig.pollingStrategy]);
+    useEffect(() => {
+        // ensure that state never gets out of sync with the config
+        setSelectedPollingRate(pollingConfig.minimumPollingFrequency);
+    }, [pollingConfig.minimumPollingFrequency]);
+
+    logDebugConsole('debug', 'Render {pollingConfig}', pollingConfig);
+    return (
+        <HeaderControlButtonWithCallout
+            buttonProps={{
+                iconName: 'DeveloperTools',
+                testId: 'scene-configure-scene-button',
+                title: t(LOC_KEYS.headerButtonTitle)
+            }}
+            calloutProps={{
+                iconName: 'DeveloperTools',
+                title: t(LOC_KEYS.calloutTitle)
+            }}
+            styles={classNames.subComponentStyles.headerControlWithCallout}
+        >
+            <Stack tokens={{ childrenGap: 8 }}>
+                <span className={classNames.calloutDescription}>
+                    {t(LOC_KEYS.calloutDescription)}
+                </span>
+
+                <Dropdown
+                    ariaLabel={t(LOC_KEYS.strategyDropdownLabel)}
+                    label={t(LOC_KEYS.strategyDropdownLabel)}
+                    selectedKey={selectedPollingStrategy}
+                    options={strategyOptions}
+                    onChange={onStrategyChange}
+                    onRenderLabel={getLabelRenderFunction(
+                        t(LOC_KEYS.strategyTooltip)
+                    )}
+                />
+                {pollingConfig.pollingStrategy !== 'Realtime' && (
```

Comment:
I still think that we should keep these pollingStraregy as enum, or import it from config schema file

Reply:
Agree. It actually is typed and comes from the schema. It's just a type instead of enum since that's how the JSON file converts to TS types.